src/app/public/modules/modal/modal.component.ts

```diff
@@ -40,10 +45,21 @@ let skyModalUniqueIdentifier: number = 0;
   ]
 })
 export class SkyModalComponent implements AfterViewInit {
-  public modalState = 'in';
-  public modalContentId: string = 'sky-modal-content-id-' + skyModalUniqueIdentifier.toString();
-  public modalHeaderId: string = 'sky-modal-header-id-' + skyModalUniqueIdentifier.toString();
 
+  /**
+   * @internal
```

Comment:
Just curious: Why are `ariaRole` and `tiledBody` internal properties? Is it because we have the corresponding configuration properties?

Reply:
Good question, I was also curious about this when I started going through the repo.

By exposing these as `@Input` properties on the component, we allow other components that talk to the modal to easily leverage the properties. I marked these as `@internal` as to not confuse consumers since a vast majority (if not all) should be utilizing the service to open and manage modals.